source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,205 @@
+.. _security:
+
+Clear Linux\* Security 
+**********************
+
+Clear Linux\*, as it is distributed and maintained upstream, aims to make 
+systemic and layered security conscious decisions, which are both
```

Comment:
Correction:  security-conscious decisions that are both peformant and practical.